playbook/roles/varnish/templates/default.vcl.j2

```diff
@@ -268,6 +268,18 @@ sub vcl_recv {
     return (pass);
   }
 
+
+  {% if varnish.recv_conditions is defined %}
+
+  # Conditions specified by variables:
+  {% for condition in varnish.recv_conditions %}
+  if (req.{{ condition.property }} {{ condition.operator }} "{{ condition.value }}") {
```

Comment:
Should property include the "req" part here too? In some rare cases there might be need to check some other properties besides req.

Reply:
ok! (my doubt was that in that context only the request would be relevant)